src/views/EdgeApplicationsFunctions/FormFields/FormFieldsEdgeApplicationsOrigins.vue

```diff
@@ -0,0 +1,90 @@
+<script setup>
+  import InputText from 'primevue/inputtext'
+  import Dropdown from 'primevue/dropdown'
+  import FormHorizontal from '@/templates/create-form-block/form-horizontal'
+  import { useField } from 'vee-validate'
+  import { computed } from 'vue'
+  import { useAccountStore } from '@/stores/account'
+
+  defineProps({
+    edgeFunctionsList: {
+      required: true,
+      type: Array
+    }
+  })
+
+  const store = useAccountStore()
+
+  const editorOptions = computed(() => {
+    return {
+      minimap: { enabled: false },
+      tabSize: 2,
+      formatOnPaste: true
+    }
+  })
+
+  const { value: name } = useField('name')
+  const { value: edgeFunctionID } = useField('edgeFunctionID')
+  const { value: args } = useField('args')
+
+  const theme = computed(() => {
+    return store.currentTheme === 'light' ? 'vs' : 'vs-dark'
+  })
+</script>
+
+<template>
+  <FormHorizontal
+    :isDrawer="true"
+    title="General"
+    description="Instantiate an edge function within your edge application."
+  >
+    <template #inputs>
+      <div class="flex flex-col sm:max-w-lg w-full gap-2">
+        <InputText
+          label="Name *"
+          name="name"
+          v-model="name"
+        />
```

Comment:
O name tem o * de campo obrigatório, mas não tem nenhuma mensagem de error?
E temos um component para campos com validação que é o **FieldText**:
```suggestion
      <FieldText
          label="Name *"
          name="name"
          :value="name"
        />
```

Reply:
Mesmo que não tenha uma mensagem de error sugiro utilizar o component FieldText, pois a ideia e atualizar os campos dos form's para esses components de field.